### ISSUE TITLE
Few updates to felmyst

### ISSUE THREAD

**Issue by CraftedRO** [TrinityCore/TrinityCore#28288]
### Description

Few updates to felmyst , no sniffs for this but for sure felmyst shoud be able to fly.

It looks way better now :D

![WoWScrnShot_092622_034027](https://user-images.githubusercontent.com/24683355/192173561-5aea23e0-3831-4fda-8f37-54eae751432e.jpg)

### SQL fix

```sql
-- Let felmyst to fly kek
UPDATE `creature_template_movement` SET `Swim` = 0, `Flight` = 1 WHERE `CreatureId` = 25038;

-- Attempt to make Demonic Vapor (Trail) trigger invisible to players
UPDATE `creature_template` SET `flags_extra`=128 WHERE `entry`=25267;

-- felmyst missing BREATH_EMOTE
DELETE FROM `creature_text` WHERE `CreatureID`=25038 AND `GroupID`=7;
INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES 
(25038, 7, 0, '%s takes a deep breath.', 41, 0, 100, 0, 0, 0, 20021, 0, 'felmyst - BREATH_EMOTE');
```

### Branch

3.3.5

### TC rev. hash/commit

7eb482ed8416

**Comment by CraftedRO** [comment]
for some reason emote still doesn't appear , probably something related to brodcasttextid

**Comment by Ovahlord** [comment]
more likey out of range. you gotta increase the text range

**Comment by Ovahlord** [comment]
check their modelIds sometimes trigger have their invisible model flipped which is super annoying because it requires scripted behavior

**Comment by CraftedRO** [comment]
Original modelid1-11686 and modelid2-1126 for entry 25267 tried to switch between but same result , however with extraflag 128 looks a lot better than without:


https://user-images.githubusercontent.com/24683355/192313871-2417ad63-0c5e-4e2b-a0a8-a62445699dd3.mp4

**Comment by xvwyh** [comment]
> check their modelIds. sometimes trigger have their invisible model flipped which is super annoying because it requires scripted behavior

I'm 99% certain that blizzard does this via probabilities. They just set 0 probability for the visible model, and leave non-0 for invisible model. My guess is that WowEdit just displays spawns with the very first defined model, so that's why they put infernals/bunnies in there - to let them be visible in the editor - but when in-game the model is selected based on probabilities, and "trigger" models get skipped due to it being 0.

TL;DR: Port the ~~probability columns for creature_template~~ creature_template_model 😩 from master branch and start using them, and forget about trigger extra flag hacks.
